src/cmd-push-container

```diff
@@ -37,35 +37,43 @@ latest_build_path = f"builds/{latest_build}/{arch}"
 metapath = f"{latest_build_path}/meta.json"
 with open(metapath) as f:
     meta = json.load(f)
-ociarchive = os.path.join(latest_build_path, meta['images']['ostree']['path'])
 
-skopeoargs = ['skopeo', 'copy']
-if args.authfile is None:
-    args.authfile = os.environ.get("REGISTRY_AUTH_FILE")
-if args.authfile is not None:
-    skopeoargs.extend(['--authfile', args.authfile])
-if args.format is not None:
-    skopeoargs.extend(['--format', args.format])
-if ":" not in args.name:
-    container_name = args.name
-    # If a specific tag wasn't requested, add a default one
-    container_tag = f"{latest_build}-{arch}"
+extensions_ociarchive = os.path.join(latest_build_path, meta['images']['extensions-container']['path'])
+if not extensions_ociarchive:
+    ociarchives = ["ostree", "extensions-container"]
```

Comment:
Here `extensions_ociarchive` will always be a string i.e. truthy.  I think you may have meant to do e.g.:

```
containers_to_push = ["ostree"]
extensions = meta['images'].get('extensions-container')
if extensions:
   containers_to_push.append('extensions-container')
```
or so?

Reply:
big woops, thanks!